app/views/admin_censor_rule/_form.html.erb

```diff
@@ -39,7 +39,22 @@
 <div class="control-group">
   <label for="censor_rule_replacement" class="control-label">Replacement</label>
   <div class="controls">
-    <%= text_field 'censor_rule', 'replacement', class: 'span6'  %>
+    <% placeholder =
+      if @censor_rule.canned_replacements.any?
+        'Select or add your own'
+      end
+    %>
+
+    <%= text_field 'censor_rule',
+                   'replacement',
+                   class: 'span6',
+                   list: 'canned-replacements',
+                   placeholder: placeholder %>
+
+    <datalist id="canned-replacements">
```

Comment:
TIL datalist

Reply:
Not seeing the dropdown arrow on Firefox.

I have to either start typing or click anywhere on the input (while it is focused) to see the options.

Also shows any previous values which have been entered into this field.

Reply:
> Not seeing the dropdown arrow on Firefox.

I'm sure they'll resolve this at some point, or FF users will get used to its behaviour for this field. Not going to put further energy into it for admin interface, but if we come across a trivial fix would be happy to consider it.

> Also shows any previous values which have been entered into this field.

Added `autocomplete: 'off'` to prevent.